src/main/java/com/cloudant/client/org/lightcouch/internal/CouchDbUtil.java

```diff
@@ -182,6 +189,26 @@ public static String readFile(String path) {
         return content.toString();
     }
 
+    public static String readURL(URL url) {
```

Comment:
I don't think this is needed if a `FileReader` is used for https://github.com/cloudant/java-cloudant/pull/132/files#r44260039